### PR TITLE
doc: typo fix

### PR DESCRIPTION
**Pull request by revolunet**
small typo fix

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| buildDuration | 24.7s | 24.6s | -67ms |
| nodeModulesSize | 41.2 MB | 41.2 MB | -74 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.js | 18.1 kB | 18.1 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.module.js | 16.5 kB | 16.5 kB | ✓ |
| main-HASH.module.js gzip | 6.36 kB | 6.36 kB | ✓ |
| webpack-HASH.module.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.module.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.js | 1.9 kB | 1.9 kB | ✓ |
| _app.js gzip | 908 B | 908 B | ✓ |
| _error.js | 12.1 kB | 12.1 kB | ✓ |
| _error.js gzip | 4.76 kB | 4.76 kB | ✓ |
| hooks.js | 12.7 kB | 12.7 kB | ✓ |
| hooks.js gzip | 4.81 kB | 4.81 kB | ✓ |
| index.js | 343 B | 343 B | ✓ |
| index.js gzip | 237 B | 237 B | ✓ |
| link.js | 8.14 kB | 8.14 kB | ✓ |
| link.js gzip | 3.49 kB | 3.49 kB | ✓ |
| routerDirect.js | 433 B | 433 B | ✓ |
| routerDirect.js gzip | 296 B | 296 B | ✓ |
| withRouter.js | 444 B | 444 B | ✓ |
| withRouter.js gzip | 294 B | 294 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.module.js | 1.79 kB | 1.79 kB | ✓ |
| _app.module.js gzip | 867 B | 867 B | ✓ |
| _error.module.js | 23.3 kB | 23.3 kB | ✓ |
| _error.module.js gzip | 8.62 kB | 8.62 kB | ✓ |
| hooks.module.js | 1.55 kB | 1.55 kB | ✓ |
| hooks.module.js gzip | 804 B | 804 B | ✓ |
| index.module.js | 319 B | 319 B | ✓ |
| index.module.js gzip | 238 B | 238 B | ✓ |
| link.module.js | 8.52 kB | 8.52 kB | ✓ |
| link.module.js gzip | 3.68 kB | 3.68 kB | ✓ |
| routerDirect.module.js | 419 B | 419 B | ✓ |
| routerDirect.module.js gzip | 294 B | 294 B | ✓ |
| withRouter.module.js | 429 B | 429 B | ✓ |
| withRouter.module.js gzip | 293 B | 293 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _buildManifest.js | 81 B | 81 B | ✓ |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManifest.module.js | 81 B | 81 B | ✓ |
| _buildManifest.module.js gzip | 61 B | 61 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Fetched pages</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| link | 3.74 kB | 3.75 kB | ⚠️ +1 B |
| link gzip | 964 B | 965 B | ⚠️ +1 B |
| index | 3.69 kB | 3.69 kB | ⚠️ +1 B |
| index gzip | 954 B | 957 B | ⚠️ +3 B |
| withRouter | 3.75 kB | 3.75 kB | ⚠️ +1 B |
| withRouter gzip | 949 B | 952 B | ⚠️ +3 B |
| Overall change |  |  | ⚠️ +3 B |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| buildDuration | 26.7s | 26.2s | -514ms |
| nodeModulesSize | 41.2 MB | 41.2 MB | -74 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.js | 18.1 kB | 18.1 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.module.js | 16.5 kB | 16.5 kB | ✓ |
| main-HASH.module.js gzip | 6.36 kB | 6.36 kB | ✓ |
| webpack-HASH.module.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.module.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.js | 1.9 kB | 1.9 kB | ✓ |
| _app.js gzip | 908 B | 908 B | ✓ |
| _error.js | 12.1 kB | 12.1 kB | ✓ |
| _error.js gzip | 4.76 kB | 4.76 kB | ✓ |
| hooks.js | 12.7 kB | 12.7 kB | ✓ |
| hooks.js gzip | 4.81 kB | 4.81 kB | ✓ |
| index.js | 343 B | 343 B | ✓ |
| index.js gzip | 237 B | 237 B | ✓ |
| link.js | 8.14 kB | 8.14 kB | ✓ |
| link.js gzip | 3.49 kB | 3.49 kB | ✓ |
| routerDirect.js | 433 B | 433 B | ✓ |
| routerDirect.js gzip | 296 B | 296 B | ✓ |
| withRouter.js | 444 B | 444 B | ✓ |
| withRouter.js gzip | 294 B | 294 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.module.js | 1.79 kB | 1.79 kB | ✓ |
| _app.module.js gzip | 867 B | 867 B | ✓ |
| _error.module.js | 23.3 kB | 23.3 kB | ✓ |
| _error.module.js gzip | 8.62 kB | 8.62 kB | ✓ |
| hooks.module.js | 1.55 kB | 1.55 kB | ✓ |
| hooks.module.js gzip | 804 B | 804 B | ✓ |
| index.module.js | 319 B | 319 B | ✓ |
| index.module.js gzip | 238 B | 238 B | ✓ |
| link.module.js | 8.52 kB | 8.52 kB | ✓ |
| link.module.js gzip | 3.68 kB | 3.68 kB | ✓ |
| routerDirect.module.js | 419 B | 419 B | ✓ |
| routerDirect.module.js gzip | 294 B | 294 B | ✓ |
| withRouter.module.js | 429 B | 429 B | ✓ |
| withRouter.module.js gzip | 293 B | 293 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _buildManifest.js | 81 B | 81 B | ✓ |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManifest.module.js | 81 B | 81 B | ✓ |
| _buildManifest.module.js gzip | 61 B | 61 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _error.js | 248 kB | 248 kB | ⚠️ +13 B |
| _error.js gzip | 66.3 kB | 66.3 kB | ⚠️ +8 B |
| hooks.html | 3.81 kB | 3.81 kB | ⚠️ +4 B |
| hooks.html gzip | 978 B | 975 B | -3 B |
| index.js | 248 kB | 248 kB | ⚠️ +13 B |
| index.js gzip | 66.6 kB | 66.7 kB | ⚠️ +110 B |
| link.js | 256 kB | 256 kB | ⚠️ +13 B |
| link.js gzip | 68.9 kB | 68.9 kB | -31 B |
| routerDirect.js | 249 kB | 249 kB | ⚠️ +13 B |
| routerDirect.js gzip | 66.8 kB | 66.7 kB | -80 B |
| withRouter.js | 249 kB | 249 kB | ⚠️ +13 B |
| withRouter.js gzip | 66.8 kB | 66.9 kB | ⚠️ +108 B |
| Overall change |  |  | ⚠️ +69 B |

</details>

</details>

**Review by timneutkens — APPROVED** [review]

File: packages/next/README.md [hunk 1]
```diff
@@ -9,7 +9,6 @@
   </a>
   <a aria-label="License" href="https://github.com/zeit/next.js/blob/canary/license.md">
     <img alt="" src=
-  "https://img.shields.io/npm/l/next.svg?style=for-the-badge&labelColor=000000">
```
Comment (timneutkens) [inline packages/next/README.md]: Looks like this was deleted 🤔

**Review by timneutkens — CHANGES_REQUESTED** [review]
Added comment

**Comment by revolunet** [comment]
my bad

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| buildDuration | 22.1s | 22s | -75ms |
| nodeModulesSize | 41.2 MB | 41.2 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.js | 18.1 kB | 18.1 kB | ⚠️ +1 B |
| main-HASH.js gzip | 6.62 kB | 6.63 kB | ⚠️ +1 B |
| webpack-HASH.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ⚠️ +1 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.module.js | 16.5 kB | 16.5 kB | ⚠️ +1 B |
| main-HASH.module.js gzip | 6.36 kB | 6.36 kB | ⚠️ +1 B |
| webpack-HASH.module.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.module.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | ⚠️ +1 B |

</details>

<details>
<summary><strong>Client Pages</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.js | 1.9 kB | 1.9 kB | ✓ |
| _app.js gzip | 908 B | 908 B | ✓ |
| _error.js | 12.1 kB | 12.1 kB | ✓ |
| _error.js gzip | 4.76 kB | 4.76 kB | -1 B |
| hooks.js | 12.7 kB | 12.7 kB | ✓ |
| hooks.js gzip | 4.81 kB | 4.81 kB | ✓ |
| index.js | 343 B | 343 B | ✓ |
| index.js gzip | 237 B | 237 B | ✓ |
| link.js | 8.14 kB | 8.14 kB | ✓ |
| link.js gzip | 3.49 kB | 3.49 kB | -1 B |
| routerDirect.js | 434 B | 433 B | -1 B |
| routerDirect.js gzip | 296 B | 296 B | ✓ |
| withRouter.js | 444 B | 444 B | ✓ |
| withRouter.js gzip | 294 B | 294 B | ✓ |
| Overall change |  |  | -1 B |

</details>

<details>
<summary><strong>Client Pages Modern</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.module.js | 1.79 kB | 1.79 kB | ✓ |
| _app.module.js gzip | 866 B | 867 B | ⚠️ +1 B |
| _error.module.js | 23.3 kB | 23.3 kB | ✓ |
| _error.module.js gzip | 8.62 kB | 8.62 kB | ✓ |
| hooks.module.js | 1.55 kB | 1.55 kB | ✓ |
| hooks.module.js gzip | 805 B | 804 B | -1 B |
| index.module.js | 319 B | 319 B | ✓ |
| index.module.js gzip | 238 B | 238 B | ✓ |
| link.module.js | 8.52 kB | 8.52 kB | ✓ |
| link.module.js gzip | 3.68 kB | 3.68 kB | ✓ |
| routerDirect.module.js | 419 B | 419 B | ✓ |
| routerDirect.module.js gzip | 294 B | 294 B | ✓ |
| withRouter.module.js | 430 B | 429 B | -1 B |
| withRouter.module.js gzip | 293 B | 293 B | ✓ |
| Overall change |  |  | -1 B |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _buildManifest.js | 81 B | 81 B | ✓ |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManifest.module.js | 81 B | 81 B | ✓ |
| _buildManifest.module.js gzip | 61 B | 61 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Fetched pages</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| link | 3.74 kB | 3.74 kB | -1 B |
| link gzip | 961 B | 964 B | ⚠️ +3 B |
| index | 3.69 kB | 3.69 kB | -1 B |
| index gzip | 953 B | 956 B | ⚠️ +3 B |
| withRouter | 3.75 kB | 3.75 kB | -1 B |
| withRouter gzip | 948 B | 951 B | ⚠️ +3 B |
| Overall change |  |  | -3 B |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| buildDuration | 23.9s | 23.3s | -614ms |
| nodeModulesSize | 41.2 MB | 41.2 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.js | 18.1 kB | 18.1 kB | -1 B |
| main-HASH.js gzip | 6.63 kB | 6.62 kB | -1 B |
| webpack-HASH.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | -1 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| main-HASH.module.js | 16.5 kB | 16.5 kB | -1 B |
| main-HASH.module.js gzip | 6.36 kB | 6.36 kB | -1 B |
| webpack-HASH.module.js | 1.53 kB | 1.53 kB | ✓ |
| webpack-HASH.module.js gzip | 746 B | 746 B | ✓ |
| Overall change |  |  | -1 B |

</details>

<details>
<summary><strong>Client Pages</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.js | 1.9 kB | 1.9 kB | ✓ |
| _app.js gzip | 908 B | 908 B | ✓ |
| _error.js | 12.1 kB | 12.1 kB | ✓ |
| _error.js gzip | 4.76 kB | 4.76 kB | ⚠️ +1 B |
| hooks.js | 12.7 kB | 12.7 kB | ✓ |
| hooks.js gzip | 4.81 kB | 4.81 kB | ✓ |
| index.js | 343 B | 343 B | ✓ |
| index.js gzip | 237 B | 237 B | ✓ |
| link.js | 8.14 kB | 8.14 kB | ✓ |
| link.js gzip | 3.49 kB | 3.49 kB | ⚠️ +1 B |
| routerDirect.js | 433 B | 434 B | ⚠️ +1 B |
| routerDirect.js gzip | 296 B | 296 B | ✓ |
| withRouter.js | 444 B | 444 B | ✓ |
| withRouter.js gzip | 294 B | 294 B | ✓ |
| Overall change |  |  | ⚠️ +1 B |

</details>

<details>
<summary><strong>Client Pages Modern</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _app.module.js | 1.79 kB | 1.79 kB | ✓ |
| _app.module.js gzip | 867 B | 866 B | -1 B |
| _error.module.js | 23.3 kB | 23.3 kB | ✓ |
| _error.module.js gzip | 8.62 kB | 8.62 kB | ✓ |
| hooks.module.js | 1.55 kB | 1.55 kB | ✓ |
| hooks.module.js gzip | 804 B | 805 B | ⚠️ +1 B |
| index.module.js | 319 B | 319 B | ✓ |
| index.module.js gzip | 238 B | 238 B | ✓ |
| link.module.js | 8.52 kB | 8.52 kB | ✓ |
| link.module.js gzip | 3.68 kB | 3.68 kB | ✓ |
| routerDirect.module.js | 419 B | 419 B | ✓ |
| routerDirect.module.js gzip | 294 B | 294 B | ✓ |
| withRouter.module.js | 429 B | 430 B | ⚠️ +1 B |
| withRouter.module.js gzip | 293 B | 293 B | ✓ |
| Overall change |  |  | ⚠️ +1 B |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _buildManifest.js | 81 B | 81 B | ✓ |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManifest.module.js | 81 B | 81 B | ✓ |
| _buildManifest.module.js gzip | 61 B | 61 B | ✓ |
| Overall change |  |  | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary | revolunet/next.js patch-2 | Change |
| - | - | - | - |
| _error.js | 248 kB | 248 kB | -13 B |
| _error.js gzip | 66.3 kB | 66.5 kB | ⚠️ +171 B |
| hooks.html | 3.81 kB | 3.81 kB | -4 B |
| hooks.html gzip | 981 B | 977 B | -4 B |
| index.js | 248 kB | 248 kB | -13 B |
| index.js gzip | 66.8 kB | 66.6 kB | -190 B |
| link.js | 256 kB | 256 kB | -13 B |
| link.js gzip | 68.9 kB | 68.8 kB | -47 B |
| routerDirect.js | 249 kB | 249 kB | -13 B |
| routerDirect.js gzip | 66.9 kB | 66.9 kB | -3 B |
| withRouter.js | 249 kB | 249 kB | -13 B |
| withRouter.js gzip | 66.7 kB | 66.6 kB | -26 B |
| Overall change |  |  | -69 B |

</details>

</details>

**Review by timneutkens — APPROVED** [review]

**Pull request merged**